ci-operator/step-registry/servicemesh/kiali/int-tests/execute/servicemesh-kiali-int-tests-execute-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+OCP_API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
+OCP_CRED_USR="kubeadmin"
+OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+
+oc login ${OCP_API_URL} --username=${OCP_CRED_USR} --password=${OCP_CRED_PSW} --insecure-skip-tls-verify=true
+hack/istio/install-bookinfo-demo.sh -c oc -n bookinfo -tg -in ${SMCP_NAMESPACE}
+sleep 120
```

Comment:
If you are waiting for something to be provisioned, could we write some code to just check for what you are waiting for every 30ish seconds to see if it is ready? This may help in instances where 120 seconds isn't enough time.

Reply:
@FilipB can we just check the Bookinfo pods status?

`oc wait --for=condition=Ready pods --all -n bookinfo --timeout 60s || true`

Reply:
Waiting just for bookinfo pods to be up is not enough. We also have to wait for some traffic history to be generated. See my comment above. I'm not sure if the proper fix for this is worth the time.

Reply:
We can mark this resolved